### ISSUE TITLE
WH1080 and similar support

### ISSUE THREAD

**Issue by lwqcz** [esphome/feature-requests#488]
**Describe the problem you have/What new integration you would like**
Implement support for cheap weather station HW like WH1080 and similar is.

**Please describe your use case for this integration and alternatives you've tried:**
Hardware like WH1080 is cheap and easy to implement but there is no direct support in ESPHome and it would be interesting for rather many people.

**Additional context**
Actual Arduino library is made by @pilotak available [here](https://github.com/pilotak/WeatherMeters).

**Comment by pilotak** [comment]
I actually have already a working code https://github.com/pilotak/WemosWeatherStation but why not have it as part of esphome

**Comment by lwqcz** [comment]
Yeah, I've noticed that :-) Your work is really impressive and I assume that as part of ESPHome it could be in many cases even better :-)

**Comment by pilotak** [comment]
that's true, "Děkuji za pochvalu"
